debezium-scripting/debezium-scripting/src/test/resources/wasm/go/filter1.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+import (
+	"github.com/andreaTP/debezium-smt-go-pdk"
+)
+
+//export process
+func process(proxyPtr uint32) uint32 {
```

Comment:
Since in this case we are leveraging, mostly, statically typed languages.
We can consider having 2 separate function signatures for `filter`(`uint32 -> bool`) and `router`(`uint32 -> uint32`).